paddle/fluid/pybind/pybind.cc

```diff
@@ -95,6 +96,9 @@ bool IsCompiledWithDIST() {
 }
 
 PYBIND11_PLUGIN(core) {
+  // Not used, just make sure cpu_info.cc is linked.
+  paddle::platform::CpuTotalPhysicalMemory();
```

Comment:
The `FLAGS_use_pinned_memory` is defined in `cpu_info.cc`, which will be erased since there is not an explicit invoke to this file. The simple fix is just use a method in the `cpu_info.cc` explicitly.

Reply:
However, maybe we should remove `cpu_info` since all the methods in it are not used when `WITH_GPU=OFF`.